src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -3,40 +3,78 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { faExchangeAlt, faFolder, faHome, faQuestionCircle, faUsers } from '@fortawesome/free-solid-svg-icons';
+import { faExchangeAlt, faFolder, faHome, faQuestionCircle, faToolbox, faUsers } from '@fortawesome/free-solid-svg-icons';
 import { routes } from '../../utils/routes';
 import { LinkItem } from '../../utils/types';
 import NavPageLinks from './NavPageLinks';
 import styles from '../../stylesheets/layouts/sidebar/sidebar.module.css';
+import { useAuth } from '../../hooks/auth.hooks';
 
 const Sidebar: React.FC = () => {
-  const linkItems: LinkItem[] = [
-    {
-      name: 'Home',
-      icon: faHome,
-      route: routes.HOME
-    },
-    {
-      name: 'Projects',
-      icon: faFolder,
-      route: routes.PROJECTS
-    },
-    {
-      name: 'Change Requests',
-      icon: faExchangeAlt,
-      route: routes.CHANGE_REQUESTS
-    },
-    {
-      name: 'Teams',
-      icon: faUsers,
-      route: routes.TEAMS
-    },
-    {
-      name: 'Info',
-      icon: faQuestionCircle,
-      route: routes.INFO
-    }
-  ];
+  let linkItems: LinkItem[];
+  const auth = useAuth();
+  if (auth.user?.role === 'ADMIN' || auth.user?.role === 'APP_ADMIN') {
```

Comment:
this is way too much duplicated code. instead you can just make the list like normal then do if (user has the right role) { links.push({admin one}) }

i guess if we don't want it last then you can just always push the info one after it

Reply:
thanks for the tip I was thinking there had to be a better way to do that.